components/Carousel.tsx

```diff
@@ -0,0 +1,49 @@
+import { FunctionComponent, useState } from "react";
+import Button from "./Button";
+import CarouselPagination from "./CarouselPagination";
+import ICarousel from "types/CarouselProps";
+
+const carouselContent = [
+  ["/CarouselMessage1.svg", "/CarouselImg1.jpg"],
+  ["/CarouselMessage2.svg", "/CarouselImg2.jpg"],
+  ["/CarouselMessage3.svg", "/CarouselImg3.jpg"],
+];
+
+const Carousel: FunctionComponent<ICarousel> = (props) => {
+  const [selectedPage, setSelectedPage] = useState(0);

```

Comment:
why 0?

Reply:
selectedPage always starts on the first position of the content array.